src/platform/plugins/shared/dashboard/public/dashboard_api/unsaved_changes_manager.ts

```diff
@@ -132,7 +130,7 @@ export function initializeUnsavedChangesManager({
         // always back up view mode. This allows us to know which Dashboards were last changed while in edit mode.
         dashboardBackupState.viewMode = viewMode;
         // Backup latest state from children that have unsaved changes
-        if (hasPanelChanges || hasControlGroupChanges) {
+        if (hasPanelChanges || hasControlGroupChanges || dashboardChanges.panels) {
```

Comment:
Nit define a const like `const hasDashboardLayoutChanges = dashboardChanges.panels` so it's easier to tell at a glance why this is a different condition from `hasPanelChanges`

Reply:
resolved in 9dbce9b410d5fb657cbb50f932019026386f0074

Also changed hasPanelChanges to hasChildrenUnsavedChanges to more specific